jobs/config.json

```diff
@@ -2224,6 +2224,13 @@
       "--mode=local"
     ], 
     "scenario": "kubernetes_e2e"
+  },
+  "ci-kubernetes-e2e-gce-gpu": {
+    "args": [
+      "--env-file=platforms/gce.env",
+      "--env-file=jobs/ci-kubernetes-e2e-gce-gpu.env"
```

Comment:
rename the actual env file to `ci-kubernetes-e2e-gce-gpu.env` :-) that's what is ranting